api/data_test.go

```diff
@@ -65,6 +67,36 @@ func TestGetCacheTimeEndpoint(t *testing.T) {
 	})
 }
 
+func TestGetCachetimeReturnsError(t *testing.T) {
```

Comment:
Small one, but it looks like we've already decided to name this `CacheTime`, rather than `Cachetime` (see line 25, for instance).